js/common.js

```diff
@@ -0,0 +1,8 @@
+const $ = (select, el = document) => el.querySelector(select);
+
+const SIGN_INFO_REGEX = {
```

Comment:
REGEX로만 표현되도 충분할 것 같아요!
```suggestion
const REGEX = {
```